tests/components/matter/test_adapter.py

```diff
@@ -3,12 +3,17 @@
 
 from typing import Any
 
+import pytest
+
 from homeassistant.components.matter.const import DOMAIN
 from homeassistant.core import HomeAssistant
 from homeassistant.helpers import device_registry as dr
 
 from .common import setup_integration_with_node_fixture
 
+# TEMP: Tests need to be fixed
+pytestmark = pytest.mark.skip("all tests still WIP")
```

Comment:
Since we're not waiting for beta 0,lets fix the tests before merging.

Reply:
We've decided to break out fixing the light and adapter tests to a separate PR. The other tests are fixed.